samples/deployment/test-network/README.md

```diff
@@ -19,85 +18,75 @@ make -C $FPC_PATH/utils/docker pull
 make -C $FPC_PATH/utils/docker build
 ```
 
-Next, we package the FPC components as docker images (building on top of `fabric-private-chaincode-ccenv`) which are deployed on our test network.
-Use `CC_ID` and `CC_PATH` to define the FPC Chaincode you want to build.
+Next, we package the FPC chaincode as a docker image which are deployed on the test-network using the chaincode as a service method.
+The following snippet builds the `echo-go` FPC chaincode. 
+Note that we export `CC_ID`, `CC_NAME`, and `CC_VER` variables, which we will need  
 
 ```bash
-cd $FPC_PATH/samples/deployment/test-network
-export CC_ID=echo
-export CC_PATH=$FPC_PATH/samples/chaincode/echo
-make build
+export CC_ID=echo-go
+export CC_NAME=fpc-echo-go
+make -C $FPC_PATH/samples/chaincode/echo-go build docker
```

Comment:
with the make like this, when I start the chaincodes, they crash. If I compile them with "GOOS=linux make -C $FPC_PATH/samples/chaincode/echo-go with_go docker" then it does.

Reply:
right, I suggest we add another link to https://github.com/hyperledger/fabric-private-chaincode?tab=readme-ov-file#fpc-playground-for-non-sgx-environments

Then, the reader will now how to run the correct `make` command

Reply:
yes, that would be ok, like what is mentioned after for ercc.

Reply:
added